### ISSUE TITLE
Support for [Forza Horizon 5]

### ISSUE THREAD

**Issue by Rainhart-D** [Z1ni/XGP-save-extractor#81]
## Game package name: Microsoft.624F8B84B80_8wekyb3d8bbwe

[Microsoft.624F8B84B80_8wekyb3d8bbwe.zip](https://drive.google.com/file/d/1ogUoEr0tPEBErwwNpLMmP02wLhNnRJ8z/view?usp=sharing)

**Comment by Z1ni** [comment]
This has now been released: https://github.com/Z1ni/XGP-save-extractor/releases

If you can confirm if the extracted save works, I'll update the readme to mark this as verified.

**Comment by Z1ni** [comment]
See https://github.com/Z1ni/XGP-save-extractor/issues/71#issuecomment-1908857982.
It seems that Forza Horizon 4 saves, which seem to use the same format, don't work when moved from the MS Store version to Steam. I suspect that the Horizon 5 saves also won't work, but I can't test that by myself, as I don't have the game.

**Comment by Xybr** [comment]
I exported my Game Pass save from FH5 using the tool, then copied onto my Steam Deck's FH5 (Steam version) under .steam/steam/userdata/[<user-id>/1551360/. I deleted the original saves then copied the Game Pass saves into the folder. Restarted the Deck into Gaming Mode, loaded FH5 and it magically picked up where my Xbox save left off. I wish they had support for Steam<->Xbox save progression, but I'll take this as a win. :)
